### PR TITLE
Flashdarks can't be used for examining eyes/mouth.

### LINKED ISSUE #28333: Flashdarks shined in your eyes still make everything bright and white.
IMMERSION RUINED.

### PR DESCRIPTION
**Pull request by Expletive**
Fixes #28333

:cl: optional name here
fix: Flashdarks can no longer be used to examine head based organs.
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/flashlight.dm [hunk 1]
```diff
@@ -57,6 +57,10 @@
 			to_chat(user, "<span class='warning'>[M] doesn't have a head!</span>")
 			return
 
+		if(flashlight_power && flashlight_power < 1)
```
Comment (lzimann) [inline code/game/objects/items/devices/flashlight.dm]: this can be just `flashlight_power < 1`

Comment (Expletive) [inline code/game/objects/items/devices/flashlight.dm]: Does (null < 1) return false?

Comment (lzimann) [inline code/game/objects/items/devices/flashlight.dm]: (null < 1) will return true.

**Comment by WJohn** [comment]
booo lazy

it should make you temporarily blind!

**Pull request merged**